metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -170,7 +170,8 @@ class MetalsLanguageServer(
   private val recentlyOpenedFiles = new ActiveFiles(time)
   private val recentlyFocusedFiles = new ActiveFiles(time)
   private val languageClient = new DelegatingLanguageClient(NoopLanguageClient)
-
+  @volatile
+  private var isImportInProcess: Boolean = false
```

Comment:
If we might end up in situation when two or more threads can modify the same variable, shouldn't `AtomicRef` (in this case `AtomicBoolean`) and CAS loops be used? At least I'm always going with Atomic* whenever I had to share some mutable value.

Reply:
Ach yeah, we might need an AtomicBoolean or something. That's my bad, I suggested volotile

Reply:
Just an idea: maybe other usages of volatile should be rethinked?